config/cass-operator/cluster-scoped/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- github.com/k8ssandra/cass-operator/config/deployments/cluster?ref=6a78a8237173d9322e6c0cab94c615b1f043a906
+
+images:
+  - name: k8ssandra/cass-operator
+    newTag: 6a78a823
```

Comment:
Do we really have a need to redefine the deployed version?

Reply:
@jsanda this was your change that I merged in. 

@burmanm I believe this has something to do with CassandraTask causing test failures due to CRD versions needing to line up with cass-operator versions, or something... I'd rather keep that discussion elsewhere though as it isn't related to this PR.